### PR TITLE
CAM-12596: improve(telemetry): improve transparency of telemetry implementation

### PR DESCRIPTION
**Pull request by yanavasileva**
Related to CAM-12596

### CODE REVIEW

**Review by tmetzke — APPROVED** [review]
Looks good, I like the added transparency 👍

File: content/introduction/telemetry.md [hunk 1]
```diff
@@ -27,6 +49,7 @@ Below you find the full list of data the reporter collects, followed by a real-w
 
 The "General Data" category contains information about the process engine:
 
+* Hashed IP address - a hashed IP address from which the data is sent, used to filter duplicated data and detect malicious access
```
Comment (casulond) [inline content/introduction/telemetry.md:52]: Another possible edit similar to the above one:

```suggestion
* Hashed IP address - the telemetry service that receives the data stores a hash of the IP address from which the data was sent to filter duplicate data and detect malicious access.
```

**Review by casulond — COMMENTED** [review]
Very nice, thank you! I think wording could be changed a bit - but only optionally.

File: content/introduction/telemetry.md (hunk 1 above)
Comment (yanavasileva) [reply content/introduction/telemetry.md]: Thank you for your feedback.

**Pull request merged**